custom_model_runner/datarobot_drum/drum/typeschema_validation.py

```diff
@@ -253,15 +260,18 @@ def validate(self, dataframe):
         )
         types[Values.DATE] = dataframe.select_dtypes("datetime").shape[1] > 0
 
-        types_present = [k for k, v in types.items() if v]
+        types_present = [str(k) for k, v in types.items() if v]
+
+        def list_of_strings(input_list):
+            return [str(el) for el in input_list]
```

Comment:
this still should be ', '.join(...) to get printed the way we want for the message using f-string